libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -525,6 +527,50 @@ void AP_DDS_Client::update_topic(geographic_msgs_msg_GeoPointStamped& msg)
     }
 }
 
+bool AP_DDS_Client::update_topic(ardupilot_msgs_msg_Status& msg)
+{
+    // Fill the new message.
+    msg.vehicle_type = static_cast<uint8_t>(AP::fwversion().vehicle_type);
+    msg.armed = AP_Notify::flags.armed;
+    msg.mode = AP::vehicle()->get_mode();
+    msg.flying = AP_Notify::flags.flying;
+    uint8_t fs_iter = 0;
+    msg.failsafe_size = 0;
+    if ( AP_Notify::flags.failsafe_radio ) {
```

Comment:
nitpick: little whitespace within the brackets

Reply:
Whitespaces removed